index.html

```diff
@@ -4,6 +4,7 @@
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link href='https://api.mapbox.com/mapbox-gl-js/v2.12.0/mapbox-gl.css' rel='stylesheet' />
+    <link rel="icon" href="/public/images/favicon.ico">
```

Comment:
```suggestion
    <link rel="icon" href="/images/favicon.ico">
```
![image](https://user-images.githubusercontent.com/72620908/227163150-9a599fbf-76ab-4e65-ae32-5a7eca1869b6.png)